vrecord

```diff
@@ -26,6 +26,7 @@ AUD_OUTLIER_THRSHLD=10
 BRNG_OUTLIER_THRSHLD=14
 FFMPEG_DECKLINK=($(brew --prefix ffmpegdecklink)/bin/ffmpeg-dl)
 FFPLAY_DECKLINK=($(brew --prefix ffmpegdecklink)/bin/ffplay-dl)
+SCRIPTDIR=$(dirname $(which "${0}"))
```

Comment:
I would suggest to move this before line 8, and to use the variable in the line 8 as well.

Reply:
Where is `${SCRIPTDIR}` used?